tema14Ianuarie/src/com/sci/bv/tema14Ianuarie/Main.java

```diff
@@ -0,0 +1,133 @@
+                    Animlist = myList.addAnimalProduct(); //Add animal product and characteristics
+                    System.out.println(
+                            "-------------------------------------------------------------\n" +
+                                    "1.Add another animal product\n" +
+                                    "0.Go back to main menu\n" +
+                                    "-------------------------------------------------------------\n" +
+                                    "\n");
+                    x = scanner.nextInt();
+                }
+                while (x == 2) {
+                    Veglist = myList1.addVegetableProduct();//Add animal product and characteristics
+                    System.out.println("-------------------------------------------------------------\n" +
+                            "2.Add another vegetable product\n" +
+                            "0.Go back to main menu\n" +
+                            "-------------------------------------------------------------\n" +
+                            "\n");
+                    x = scanner.nextInt();
+                }
+
+            }
+            while (x == 2) {
+                System.out.println("-------------------------------------------------------------\n" +
+                        "ID   Quantity   Product's name      Optimal Temperature    Price    Validity    Weight\n");
+                myList.displayProduct(); //Display animal products and characteristics
+                System.out.println("-------------------------------------------------------------\n" +
+                        "ID   Quantity   Product's name      Vitamins List    Price    Validity    Weight\n");
+                myList1.displayProduct();//Display vegetable products and characteristics
+                System.out.println("Please write the ID product:");
+                int ID = scanner.nextInt();
+                System.out.println("Please write the quantity:");
+                int Qty = scanner.nextInt();
+                for (AnimalProduct a : Animlist) {
+                    if (ID == a.getProductID()) { // Verify if the given product's ID is on stock
+                        if (Qty <= a.getProductQty() && a.getProductQty() != 0) {//Verify if is in stock the given quantity
+                            System.out.println("The order is valid.");
+                            System.out.println("Plese write the order date (yyyyMMdd):");
+                            int day = scanner.nextInt();
+                            Sales.setDate(day);
+                            Sales.setID(a.getProductID());
+                            Sales.setQty(Qty);
+                            DailyS.add(new DailySales(a.getProductID(), Qty, day));//Save the sale in Daily Sales
+                            for (int y = 1; y <= Qty; y++) {
+                                a.remove(); //Updating the quantity of products left in stock
```

Comment:
you could just create the method "public void removeQuantity(int qty)" in Animal and Vegetable product and not have to deal with this for loop here.